### ISSUE TITLE
Dynamic row height problem - datatable-scroller does not resize

### ISSUE THREAD

**Issue by FlacosFlace** [swimlane/ngx-datatable#966]
**I'm submitting a ...**  (check one with "x")
```
[ ] bug report => search github for a similar issue or PR before submitting
[x ] feature request
[ ] support request => Please do not submit support request here, post on Stackoverflow or Gitter
```

**Current behavior**
I use flexLayout to change row height. When window inner width is less than 960px row height change from 80px to 230px.  

    getRowHeight(row) {
        if (window.innerWidth < 960) {
            return 230;
        }
        else {
            return 80;
        }
    }

However the datatable-scroller does not resize height even if i call datatable.recalculate. Now when I change row height from 80px to 230px row change height but display under next rows. 

**Expected behavior**
The datatable-scroller should resize height.

**Reproduction of the problem**

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**
Latest PhpStorm, Angular CLI

* **Table version:** 10.1.0

* **Angular version:** 4.3.6

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by bluecaret** [comment]
I seem to be having the same issue. Anyone find a solution to this?

**Comment by FlacosFlace** [comment]
Not really, in the end I disturbed virtual scroll. Now works as expected.

**Comment by ArfanMirza** [comment]
any luck ? Anyone find solution ?

**Comment by FlacosFlace** [comment]
Think in new version everythings ok.

**Comment by minzdrav** [comment]
It's not ok in latest version.

**Comment by wizcas** [comment]
just checked and it's still not ok
